tests/python_tests/test_preemption.py

```diff
@@ -109,12 +109,12 @@ def test_preemption_with_multinomial(tmp_path, dynamic_split_fuse):
     ref_texts=get_current_plarform_ref_texts({
         "linux": [
             [
-                "\nI've seen this expression used too many times without making sense.\nAs an AI engineer, and as a scientist, we should make everything easier"
+                " Buzzfeed  ESPN  CNBC  MSNBC  CBS\nFox News is on top of the list.\nIf a news station tries to afford real estate"
             ],
             [
-                " position of the Z-shaped groove?\n0.41\nWhat is the current position of the Z-shaped groove?\n0.11\n",
-                " status of all of this? I can't stop thinking about it.\nIt's been a while since I've seen it. I found it a",
-                " status of your blog? Do you accept feedback?\nYes, I’m happy to accept feedback at this time (I’m a"
+                " condition of the leg?\nIt's been quite a while since I've seen it, so I didn't really know if it was good or bad",
```

Comment:
A bit strange refs..

Reply:
Multinomial tests for preemption fail on master, so perhaps there's something wrong going on with it and we get strange outputs.

Reply:
test_preemption.py passed here https://github.com/openvinotoolkit/openvino.genai/actions/runs/10056169378/job/27794414983?pr=666. But I didn't do anything specific.